test/integration_test/aws/run.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Chaos Mesh Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+cur=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+cd $cur
+
+# wait localstash pod status to running
+for ((k=0; k<30; k++)); do
+    not_run_num=`kubectl get pods -l app.kubernetes.io/name=localstack --no-headers | grep -v Running | wc -l`
+
+    if [ $not_run_num == 0 ]; then
+        break
+    fi
+
+    sleep 3
+done
+
+kubectl port-forward svc/localstack 4566:4566 &
+
+NODE_PORT=$(kubectl get --namespace default -o jsonpath="{.spec.ports[0].nodePort}" services localstack)
+NODE_IP=$(kubectl get nodes --namespace default -o jsonpath="{.items[0].status.addresses[0].address}")
+LOCALSTACK_SERVER="http:\/\/$NODE_IP\:$NODE_PORT"
+
+aws configure set aws_access_key_id test
+aws configure set aws_secret_access_key test
+aws configure set default.region us-east-1
+aws configure set default.output_format text
+
+echo "run ec2 instance, and the state is pending, will switch to running later"
+aws --endpoint-url=http://127.0.0.1:4566 ec2 run-instances --image-id ami --count 1 --instance-type t2.micro --key-name test > run_instance.log
+check_contains "pending" run_instance.log
```

Comment:
I prefer to use `jq` select the specific field from the output json, then make an assertion with `test`, for example:

```bash
test $(aws --endpoint-url=http://127.0.0.1:4566 ec2 run-instances --image-id ami --count 1 --instance-type t2.micro --key-name test | jq -rM '.Instances[0].State.Name') = "pending"
```

Reply:
done in 69dcf65